### PR TITLE
FUSETOOLS2-1160 - fixing register didact tutorial

### PR DESCRIPTION
**Pull request by bfitzpat**
Signed-off-by: Brian Fitzpatrick <bfitzpat@redhat.com>

### CODE REVIEW

**Review by apupier — CHANGES_REQUESTED** [review]
A non-regression test seems very important. it was spotted that a test was missing during implementation of the feature, it wasn't provided. A regression has been hit. It is showing that it is a really important test.

**Comment by bfitzpat** [comment]
> A non-regression test seems very important. it was spotted that a test was missing during implementation of the feature, it wasn't provided. A regression has been hit. It is showing that it is a really important test.

Your statement here is unclear. Please clarify. The tests that have failed are ui-tests.

**Comment by apupier** [comment]
> > A non-regression test seems very important. it was spotted that a test was missing during implementation of the feature, it wasn't provided. A regression has been hit. It is showing that it is a really important test.
> 
> Your statement here is unclear. Please clarify. 

The didact tutorial contribution was spotted as a feature for which [it is important to provide an automated test](https://issues.redhat.com/browse/FUSETOOLS2-473). It wasn't done. The regression is demonstrating that it is a sensible point for which a non-regression test would be very useful.


> The tests that have failed are ui-tests.

My comment is unrelated to the build failure.

**Comment by bfitzpat** [comment]
Feel free to provide the regression test in a separate PR.

File: src/extension.ts [hunk 1]
```diff
@@ -494,11 +493,10 @@ async function registerTutorialWithDidact(context: vscode.ExtensionContext, name
 		const didactExt: any = vscode.extensions.getExtension(extensionId);
 		if (didactExt) {
 			const commandId: string = 'vscode.didact.register';
-			const tutorialPath: string = path.join(context.extensionPath, extpath);
-			const tutorialUri: vscode.Uri = vscode.Uri.parse(`file://${tutorialPath}`);
+			const tutorialUri: vscode.Uri = vscode.Uri.file(context.asAbsolutePath(extpath));
 
 			// then pass name, uri, and category
```
Comment (apupier) [inline src/extension.ts:498]: it is no more the uri which is passed

Comment (bfitzpat) [reply src/extension.ts:498]: Regardless of the object being passed, the object itself BECOMES a URI object for the tutorial, so I think the name is still appropriate for the variable so I'm not sure why you're arguing over this point.

**Comment by bfitzpat** [comment]
Looks like we can close this in favor of https://github.com/camel-tooling/vscode-camelk/pull/803?

**Comment by apupier** [comment]
agree

**Pull request closed without merging**